libs/model/package.json

```diff
@@ -63,6 +63,7 @@
     "zod": "^3.22.4"
   },
   "devDependencies": {
+    "ethers": "5.7.2",
```

Comment:
Please don't re-add ethers to the codebase. Any Web3 operations can be done entirely with Viem. In my SignIn PR I have an example for creating a wallet. The one issue is the Viem accounts are not directly compatible with Canvas SIWESigner but we can easily create a utility in evm-protocols.

Reply:
this is for testing only, but I agree, it should be encapsulated in the protocols lib... can you share the sample code I can use to sign viem with canvas?